.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+  }
```

Comment:
{
  "block_hash": "2dfa70ad70b1d62945c2e365a8527cc29f360211057090b05e7b16cd842c98aa",
  "block_height": 2697848,
  "block_index": 802,
  "hash": "c1bd1f9a610c52fdab9b8f3e2823d25b7416f6f36989d85365c154e0a24b3b3e",
  "hex": "0100000000010127a64261dbe5aca15ad36e42f9a45fe8ae8c1eefbd2097a33e077f4581efd1600100000000ffffffff02001bb7000000000017a9144877445137251b208672483254cd10a9a3221cd187a0cd0301000000001600149ce01e71766bcac7cc6532dc906937dfa1df398302483045022100ad7f43caddaa41c7521e80190619d106f7f006813dcf6880bf1bdf5ec884590302200665703910877eed6191c9d60ad39ac80d532c604965d2c990df8df5d14299e8012102901ab2afb3fa5be9347c5bdee45bf9c0b78b3e3eae8c127f9e30e08d1df9586800000000",
  "addresses": [
    "MEWKm3o2yAFaXi5b1MG6QSsNsC2txbEsod",
    "ltc1qnnspuutkd09v0nr9xtwfq6fhm7sa7wvrwxm24a"
  ],
  "total": 29026464,
  "fees": 710,
  "size": 224,
  "vsize": 142,
  "preference": "low",
  "relayed_by": "95.217.104.124:9333",
  "confirmed": "2024-06-05T20:09:02Z",
  "received": "2024-06-05T20:08:16.03Z",
  "ver": 1,
  "double_spend": false,
  "vin_sz": 1,
  "vout_sz": 2,
  "confirmations": 122,
  "confidence": 1,
  "inputs": [
    {
      "prev_hash": "60d1ef81457f073ea39720bdef1e8caee85fa4f9426ed35aa1ace5db6142a627",
      "output_index": 1,
      "output_value": 29027174,
      "sequence": 4294967295,
      "addresses": [
        "ltc1qnnspuutkd09v0nr9xtwfq6fhm7sa7wvrwxm24a"
      ],
      "script_type": "pay-to-witness-pubkey-hash",
      "age": 2697656,
      "witness": [
        "3045022100ad7f43caddaa41c7521e80190619d106f7f006813dcf6880bf1bdf5ec884590302200665703910877eed6191c9d60ad39ac80d532c604965d2c990df8df5d14299e801",
        "02901ab2afb3fa5be9347c5bdee45bf9c0b78b3e3eae8c127f9e30e08d1df95868"
      ]
    }
  ],
  "outputs": [
    {
      "value": 12000000,
      "script": "a9144877445137251b208672483254cd10a9a3221cd187",
      "addresses": [
        "MEWKm3o2yAFaXi5b1MG6QSsNsC2txbEsod"
      ],
      "script_type": "pay-to-script-hash"
    },
    {
      "value": 17026464,
      "script": "00149ce01e71766bcac7cc6532dc906937dfa1df3983",
      "addresses": [
        "ltc1qnnspuutkd09v0nr9xtwfq6fhm7sa7wvrwxm24a"
      ],
      "script_type": "pay-to-witness-pubkey-hash"
    }
  ]
}

Reply:
0x704b058d86b08f782B6B986E1352853d6Bb7133C